src/main/kotlin/no/nav/helse/domene/ytelse/infotrygd/InfotrygdService.kt

```diff
@@ -0,0 +1,115 @@
+package no.nav.helse.domene.ytelse.infotrygd
+
+import arrow.core.Either
+import arrow.core.Try
+import arrow.core.flatMap
+import no.nav.helse.domene.Fødselsnummer
+import no.nav.helse.domene.ytelse.domain.Beregningsgrunnlag
+import no.nav.helse.domene.ytelse.domain.InfotrygdSak
+import no.nav.helse.domene.ytelse.domain.InfotrygdSakOgGrunnlag
+import no.nav.helse.oppslag.infotrygd.InfotrygdSakClient
+import no.nav.helse.oppslag.infotrygdberegningsgrunnlag.InfotrygdBeregningsgrunnlagClient
+import no.nav.helse.probe.DatakvalitetProbe
+import org.slf4j.LoggerFactory
+import java.time.LocalDate
+
+class InfotrygdService(private val infotrygdBeregningsgrunnlagClient: InfotrygdBeregningsgrunnlagClient,
+                       private val infotrygdSakClient: InfotrygdSakClient,
+                       private val probe: DatakvalitetProbe) {
+
+    companion object {
+        private val log = LoggerFactory.getLogger(InfotrygdService::class.java)
```

Comment:
Jeg tror ikke det er nødvendig å legge loggen til companion object, siden LoggerFactory er static og cache alle logene basert på key'en som du definere..

Reply:
Det er mest en vanesak for min del, i java skriver man typisk private static final.... Om loggen først er static så tenker jeg det like godt kan være en companion, for da vil ikke propertyen bli instansiert hver gang